templates/vhost/_ssl.erb

```diff
@@ -4,22 +4,22 @@
   SSLEngine on
   SSLCertificateFile      "<%= @ssl_cert %>"
   SSLCertificateKeyFile   "<%= @ssl_key %>"
-  <%- if @ssl_chain -%>
+  <%- if @ssl_chain && @ssl_chain != '' -%>
```

Comment:
\*nod\*
in that case, that template, well, our templates in general, need some fixing ;)